ergotree-ir/src/mir/header.rs

```diff
@@ -20,6 +22,47 @@ pub struct PreHeader {
     pub votes: Vec<u8>,
 }
 
+/// Represents data of the block header available in Sigma propositions.
+#[derive(PartialEq, Eq, Debug, Clone)]
+pub struct Header {
+    /// Block version, to be increased on every soft and hardfork.
+    pub version: u8,
+    /// Bytes representation of ModifierId of this Header
+    pub id: Vec<u8>,
```

Comment:
Looks like a good fit for a new type. `BlockId(Digest32)` should be a good fit. See `TxId` for example. The same goes to `parent_id`.